vendor/ember-cli/test-support-suffix.js

```diff
@@ -0,0 +1,9 @@
+// set `runningTests` variables
+runningTests = true;
+
+/*
+  This file overrides a file built into ember-cli's build pipeline and prevents
+  this built-in `Testem.hookIntoTestFramework` invocation:
+
+  https://github.com/ember-cli/ember-cli/blob/v3.20.0/lib/broccoli/test-support-suffix.js#L3-L5
+*/
```

Comment:
The reason this override is needed, is because the built-in ember-cli invocation of `Testem.hookIntoTestFramework()` happens _during_ `test-support.js` evaluation, which (when we are leveraging `Embrodier` or `ember-auto-import`) `QUnit` is not present (since its not eagerly evaluated, we only are guaranteed it is present when we import it). As such the `Testem.hookIntoTestFramework` invocation has been moved into our `addon-test-support/index.js` which is only ran once `QUnit` is actually defined/available.